Src/LinearSolvers/MLMG/AMReX_MLEBABecLap_2D_K.H

```diff
@@ -360,7 +356,7 @@ void mlebabeclap_ebflux (int i, int j, int k, int n,
             +       (            - gy*sy - gx*gy*sx*sy) * x(i ,jj,k,n)
             +       (                    + gx*gy*sx*sy) * x(ii,jj,k,n) ;
 
-        Real dphidn = dhx*(phib-phig)/dg;
```

Comment:
Is this right?  If `dhx` is removed, this will have the wrong unit.

Reply:
The scaling is moved inside dg.

Reply:
Sorry I was to quick yesterday. Thinking it through thoroughly I understood the problem. It is now fixed. Thanks!